test.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+# TODO: rewrite to cargo-make or something like that to only rebuild the sysroot when needed?
```

Comment:
cg_clif has a `config.txt` file with a `keep_sysroot` option that I can quickly enable.